worlds/witness/__init__.py

```diff
@@ -67,20 +69,31 @@ def generate_basic(self):
         items_by_name = dict()
         for item in self.items.ITEM_TABLE:
             witness_item = self.create_item(item)
-            if item not in self.items.EVENT_ITEM_TABLE:
+            if item in self.items.PROGRESSION_TABLE:
                 pool.append(witness_item)
                 items_by_name[item] = witness_item
+            elif item in self.items.EXTRA_AMOUNTS:
+                for i in range(0, self.items.EXTRA_AMOUNTS[item]):
+                    pool.append(witness_item)
 
-        # Put good item on first check
-        random_good_item = self.world.random.choice(self.items.GOOD_ITEMS)
-        first_check = self.world.get_location(
-            "Tutorial Gate Open", self.player
-        )
-        first_check.place_locked_item(items_by_name[random_good_item])
-        pool.remove(items_by_name[random_good_item])
+        less_junk = 0
+
+        # Put good item on first check if symbol shuffle is on
+        # symbols = is_option_enabled(self.world, self.player, "shuffle_symbols")
+        symbols = True
+
+        if symbols:
```

Comment:
What's happenin' here?

Reply:
Specifically, I think this gave an error if the corresponding option doesn't actually exist in the dictionary of option names to option classes